roles/cs.nginx/templates/nginx.logrotate

```diff
@@ -0,0 +1,25 @@
+/var/log/nginx/*.log {
+
+        compress
+        compresscmd /usr/bin/zstd
+        uncompresscmd /usr/bin/unzstd
+        compressext .zstd
+        compressoptions -19
+
+        rotate 52
```

Comment:
This is not crucial, but now that we install a custom conf, it'd be convenient to have this (and maybe other parameters too) configurable per-project.